src/sourmash/commands.py

```diff
@@ -191,6 +194,9 @@ def compare(args):
                     y.append('{}'.format(similarity[i][j]))
                 w.writerow(y)
 
+    if size_may_be_inaccurate:
+        notify("WARNING: size estimation for at least one of these sketches may be inaccurate. ANI values cannot be generated for these comparisons.")
```

Comment:
what do you think about changing this error message (and the other similar ones) to "ANI values will be set to 0 for these comparisons" ?

Reply:
definitely happy to do that here in `compare`, bc I set them to 0 in `compare`. For `search`, `gather`, `prefetch`, etc, the ANI values are empty, rather than 0 -- so I was thinking of leaving the warning as-is for those. Thoughts?

(explanation: 0 is a valid ANI result, so I didn't want to return zeroes when we want to ignore ANI. However, `compare` requires a number, so I had to go with 0. Open to thoughts/better practices.)

Reply:
no, I like that very much! Maybe "ANI values will not be reported for these comparisons" or even better "ANI values will be empty in output files" or something like that? I just like knowing it's not zero, or NA, or whatever.

Reply:
I like "will not be reported"!